command/shell.go

```diff
@@ -154,26 +160,9 @@ func (c *shellExec) Execute(ctx context.Context,
 			c.Shell, c.Script)
 	}
 
-	if err = localCmd.Start(ctx); err != nil {
-		logger.System().Debugf("error spawning shell process: %v", err)
-		return err
-	}
-
-	pid := localCmd.GetPid()
-
-	logger.System().Debugf("spawned shell process with pid %d", pid)
```

Comment:
Same question here about logging the pid to the task's logs.

Reply:
this happens in the process constructor.